addons/website/static/src/snippets/s_searchbar/search_bar.js

```diff
@@ -133,6 +133,7 @@ export class SearchBar extends Interaction {
         }
         this.hasDropdown = !!res;
         prevMenuEl?.remove();
+        this.updateContent(); // TODO Fix-me / Move-me
```

Comment:
Instead of this, I'd rather:
```js
            this.keepLast.add(this.waitFor(this.fetch())).then(this.waitFor(this.render));
```
in`onInput`

Reply:
This modification doesn't solve the issue from the tour. The class `show` is not applied on the `.s_searchbar_input` element and therefor the tour still fails at the last step.

Reply:
Then I guess it means it is also missing elsewhere